_docs/020-getting-started-on-gke.md

```diff
@@ -104,21 +103,26 @@ watch -n 1 kubectl get po,deploy --namespace riff-system
 ```
 
 ```
-NAME                                                READY     STATUS    RESTARTS   AGE
-po/demo-riff-function-controller-7d959dbf4f-p7pnz   1/1       Running   0          5m
-po/demo-riff-http-gateway-666bb96d6c-hzmvn          1/1       Running   0          5m
-po/demo-riff-topic-controller-dcf76d565-mw6th       1/1       Running   0          5m
-po/transport-kafka-68b986865b-6tsbk                 1/1       Running   3          11m
-po/transport-zookeeper-85fc6df85c-v6kxx             1/1       Running   0          11m
+NAME                                                   READY     STATUS    RESTARTS   AGE
+po/control-riff-function-controller-7d959dbf4f-p7pnz   1/1       Running   0          5m
+po/control-riff-http-gateway-666bb96d6c-hzmvn          1/1       Running   0          5m
+po/control-riff-topic-controller-dcf76d565-mw6th       1/1       Running   0          5m
+po/transport-kafka-68b986865b-6tsbk                    1/1       Running   3          11m
+po/transport-zookeeper-85fc6df85c-v6kxx                1/1       Running   0          11m
 
-NAME                                   DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-deploy/demo-riff-function-controller   1         1         1            1           5m
-deploy/demo-riff-http-gateway          1         1         1            1           5m
-deploy/demo-riff-topic-controller      1         1         1            1           5m
-deploy/transport-kafka                 1         1         1            1           11m
-deploy/transport-zookeeper             1         1         1            1           11m
+NAME                                      DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deploy/control-riff-function-controller   1         1         1            1           5m
+deploy/control-riff-http-gateway          1         1         1            1           5m
+deploy/control-riff-topic-controller      1         1         1            1           5m
+deploy/transport-kafka                    1         1         1            1           11m
+deploy/transport-zookeeper                1         1         1            1           11m
 ```
 
+### install the current riff CLI tool
+
+Starting with the 0.0.4 version we provide a CLI tool written in Go that makes it easy to create and deploy functions.
```

Comment:
The release that first included the goCLI doesn't really matter

Reply:
fixed in 13fbdd8